ecleankernel/layout/std.py

```diff
@@ -48,6 +48,9 @@ class StdLayout(ModuleDirLayout):
         '.lz',
         '.xz',
 
+        # refind
+        '.png',
```

Comment:
Also here and below. Let's use the new coding style for all new code.

Reply:
I think I got them all now.